### ISSUE TITLE
NEW term? acyl–coenzyme A diphosphatase

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#21216]
Do we have a term for this activity?

FIT2 (FITM2) is an acyl–coenzyme A diphosphatase crucial for endoplasmic reticulum homeostasis
  | PMID: 32915949

**Comment by deustp01** [comment]
No, AFAIK, but there is a [Rhea reaction](https://www.rhea-db.org/rhea/65564)

**Comment by alanbridge** [comment]
Hi Peter and Val, 

hope you are well, try the parent reaction as a base: 

https://www.rhea-db.org/rhea/50044

an acyl-CoA + H2O = adenosine 3',5'-bisphosphate + an acyl-4'-phosphopantetheine + 2 H(+)

All the best, Alan

**Comment by hdrabkin** [comment]
Hi @ValWood 
Since in the paper, 'Oleoyl-CoA and arachidonoyl-CoA were the only detected fatty acyl-CoAs, and levels of both species were increased in FIT2-KO cells compared with WT cells (Fig. 3 E), consistent with both being substrates for the enzyme" I'm more inclined to make the more generic reaction suggested by @alanbridge 
Interestingly, the paper you cite is used by RHEA for the more specific term.
What would you think is good parent term?

**Comment by ValWood** [comment]
> Interestingly, the paper you cite is used by RHEA for the more specific term.

I think it is the only paper, was a recent novel gene characterization in my unknown set ;)

**Comment by alanbridge** [comment]
Funnily enough (and from a completely different angle) we just designated ndx-8 in C. elegans for an update in UniProt (https://www.uniprot.org/uniprot/Q9NA25). It has this text annotation:

 "Coenzyme A diphosphatase which mediates the cleavage of CoA into 3',5'-ADP and 4'-phosphopantetheine" but has no Rhea. 

We also tagged the related protein NUDT15 in Arabidopsis (https://www.uniprot.org/uniprot/Q8GYB1), which has again no Rhea but this comment: 

"Coenzyme A diphosphatase which mediates the cleavage of oxidized CoA. Can use malonyl-CoA, hexanoyl-CoA, lauroyl-CoA, myristoyl-CoA and palmitoyl-CoA as substrates, but not isobutyryl-CoA or propionyl-CoA".

Maybe one for PAINTing? Pinging @marcfeuermann !
